enterprise/management/commands/email_drip_for_missing_dsc_records.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+"""
+Django management command for sending an email to learners with missing DataSharingConsent records.
+"""
+import logging
+from datetime import date, timedelta
+
+from django.core.management import BaseCommand
+
+from consent.models import DataSharingConsent, ProxyDataSharingConsent
+from enterprise import utils
+from enterprise.models import EnterpriseCourseEnrollment
+
+LOGGER = logging.getLogger(__name__)
+
+PAST_NUM_DAYS = 1
+
+
+class Command(BaseCommand):
+    """
+    Django management command for sending an email to learners with missing DataSharingConsent records
+    """
+
+    def handle(self, *args, **options):
+        """
+        Management command for sending an email to learners with a missing DSC. Designed to run daily.
+
+        Example usage:
+           $ ./manage.py email_drip_for_missing_dsc_records
+        """
+        past_date = date.today() - timedelta(days=PAST_NUM_DAYS)
+        enterprise_course_enrollments = EnterpriseCourseEnrollment.objects.select_related(
+            'enterprise_customer_user'
+        ).filter(created__date=past_date)
```

Comment:
I think we should apply `__gt` filter here otherwise records of a single day will be fetched.

Reply:
We want to fetch only for one day to avoid duplicate emails

Reply:
We want records for a particular day ... to avoid duplicate emails